docker/puppetdb/docker-entrypoint.d/10-analytics.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+if [ ! -z "${PUPPERWARE_DISABLE_ANALYTICS}" ]; then
+    echo "($0) Pupperware analytics disabled; skipping metric submission"
+    exit 0
+fi
+
+# See: https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters
+# Tracking ID
+tid=UA-132486246-1
+# Application Name
+an=puppetdb
+# Application Version
+av=$PUPPETDB_VERSION
+# Anonymous Client ID
+_file=/var/tmp/pwclientid
+cid=$(cat $_file 2>/dev/null || (cat /proc/sys/kernel/random/uuid | tee $_file))
+# Event Category
+ec=${PUPPERWARE_ANALYTICS_STREAM:-dev}
+# Event Action
+ea=start
+
+_params="v=1&t=event&tid=${tid}&an=${an}&av=${av}&cid=${cid}&ec=${ec}&ea=${ea}"
+_url="http://www.google-analytics.com/collect?${_params}"
+
+echo "($0) Sending metrics ${_url}"
+curl --silent --show-error -X POST -H "Content-Length: 0" $_url 1> /dev/null
```

Comment:
The endpoint returns some garbage text, like `GIF89a�����,D;` (seriously).

I wanted to silence that but I can't distinguish between errors and normal output in this case.
I tried silencing everything __except__ errors but I couldn't get it to work.

I even tried looking up WTF is being returned from this endpoint and why (did I curl wrong?) but couldn't find anything 🤷‍♂️ 

So I just did my best and added `1> /dev/null` to get rid of that weird line in the logs, but this means if the curl ever fails we wouldn't see it (why doesn't it output to stderr?). 

I figured it was fine for now.

Reply:
I think so, but lemme try again 🤞

Reply:
Yep, this is what I needed, in addition to `--output /dev/null`, gives us the exact behavior we want.

Updated.